berrynet/client/camera.py

```diff
@@ -1,153 +1,150 @@
 #!/usr/bin/python3
-#
-# Copyright 2018 DT42
-#
-# This file is part of BerryNet.
-#
-# BerryNet is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# BerryNet is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with BerryNet.  If not, see <http://www.gnu.org/licenses/>.
+"""
```

Comment:
We will keep license description in the comment header instead of module docstring.

Module docstring is for providing the high-level description of the module.

Example: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/accumulate_n_benchmark.py